style.css

```diff
@@ -480,3 +482,313 @@ h1 {
+  }
+}
+
+@media (min-width: 768px) {
+  body {
+    min-width: 996px;
+  }
+
+  .container-nav {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .logo {
+    padding-left: 150px;
+  }
+
+  .headline-section {
+    background-image: url(images/background-header-web.svg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding-left: 220px;
+    background-color: white;
+    border-bottom-left-radius: 150px;
+    padding-right: 150px;
+    padding-top: 559px;
+    padding-bottom: 250px;
+    width: 100%;
+    margin-top: 62px;
+    margin-bottom: 42px;
+  }
+
+  .headline-section h1 {
+    font-size: 48px;
+    line-height: 60px;
+    font-weight: 700;
+    padding-right: 400px;
+    min-width: 400px;
+  }
+
+  .headline-section div {
+    position: absolute;
+    top: 300px;
+  }
+
+  .headline-section p {
+    margin-block-start: 0;
+  }
+
+  .about-me {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    width: 447px;
+    padding: 12px 0;
+    margin: 0;
+  }
+
+  .project {
+    display: flex;
+  }
+
+  #projects {
+    width: 80%;
+    margin-left: 142px;
+  }
+
+  .first-project-image {
+    display: none;
+  }
+
+  .project-image-web {
+    width: 100%;
+    height: 100%;
+  }
+
+  .project-content {
+    width: 50%;
+    position: relative;
+    top: -33px;
+  }
+
+  #first-project {
+    flex-direction: row;
+  }
+
+  #second-project {
+    flex-direction: row-reverse;
+  }
+
+  #third-project {
+    flex-direction: row;
+  }
+
+  #fourth-project {
+    flex-direction: row-reverse;
+  }
+
+  .project-heading {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+  }
+
+  .description {
+    padding-bottom: 10px;
+  }
+
+  .content-text {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 66px 120px;
+    border-top-right-radius: 120px;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    width: 45%;
+  }
+
+  .about-web {
+    display: flex;
+    flex-direction: column;
+    padding-left: 48px;
+  }
+
+  #about-section {
+    border-top-right-radius: 120px;
+    width: 100%;
+    padding-top: 0;
+    padding-bottom: 102px;
+  }
+
+  .language-container {
+    padding: 0 48px;
+  }
+
+  .language-header-container {
+    padding: 10px 0;
+  }
+
+  .my-language-container {
+    display: inline;
+    margin-right: 20px;
+    background-color: #f7f7f9;
+    height: 90px;
+    width: 90px;
+  }
+
+  .my-language {
+    display: flex;
+    margin-top: 0;
+  }
+
+  .frameworks,
+  .skills {
+    padding: 15px 0;
+  }
+
+  .my-language-text {
+    padding-left: 0;
+    margin: 0;
+  }
+
+  #contact-me {
+    display: flex;
+    justify-content: center;
+    background-image: url(./Assets/background-images/contact-form-background-image-web.svg);
+    background-size: contain;
```

Comment:
- [ ] Your design looks great :+1: However,  From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see, your contact me section background image is not showing on desktop view :blush: 

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-03-30 10-42-47](https://user-images.githubusercontent.com/92303891/228813618-9bbfe953-61c2-4783-8702-df50e9f6e549.png) | ![image](https://user-images.githubusercontent.com/92303891/228813875-69f28fa1-c42f-40d4-b73a-78062962a1c4.png)|